src/components/SummaryKeyComponents/AbstractComponent.js

```diff
@@ -0,0 +1,45 @@
+import { getPaperAbstract } from "../../services/AbstractService";
+import {
+  EventHandlerService,
+  PDFLEvents,
+} from "../../services/EventHandlerService";
+
+/**
+ * Class representing the Abstract Accordion item
+ * @property {Object} components object that holds DOM elements that represent this component, as well as component's context
+ * @property {HTMLElement} components.abstractText html p element that represents the text of the accordion item
+ */
+class AbstractComponent {
```

Comment:
Not really a meaningful name if you think about it, I know its long and ugly but something along the lines of AbstractAccordionItemComponent is maybe better?